`GetPictureCells` and `GetPictures` functions don't support images with absolute path targets

**Description**

The `GetPictureCells` and `GetPictures` functions don't appear to support images where the target attribute in the XML structure uses absolute paths. 

For example the following `xl/drawings/_rels/drawing1.xml.rels` file which contains a Relationship with a Target attribute using an absolute path of `"/xl/media/image1.jpg"` does not work.

```xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<Relationships xmlns="http://schemas.openxmlformats.org/package/2006/relationships">
    <Relationship Id="rId1"
        Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/image"
        Target="/xl/media/image1.jpg" />
</Relationships>
```

When the Target attribute is using a relative path of `"../media/image1.jpg"` it does work.

```xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<Relationships xmlns="http://schemas.openxmlformats.org/package/2006/relationships">
    <Relationship Id="rId1"
        Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/image"
        Target="../media/image1.jpg" />
</Relationships>
```

**Steps to reproduce the issue:**

Use the attached `TestGetPictureAbsolutePath.xls` file which contains a single floating image and the following Go test which demonstrates a failure to detect _any_ images when using `GetPictureCells`.

```go
func TestGetPictureAbs(t *testing.T) {
	f, err := OpenFile(filepath.Join("test", "TestGetPictureAbsolutePath.xlsx"))
	assert.NoError(t, err)

	cells, err := f.GetPictureCells("Sheet1")
	assert.NoError(t, err)
	assert.Equal(t, []string{"A1"}, cells)
	assert.NoError(t, f.Close())

	pics, err := f.GetPictures("Sheet1", "A1")
	assert.NoError(t, err)
	assert.Len(t, pics, 1)
}
```

* [TestGetPictureAbsolutePath.xlsx](https://github.com/user-attachments/files/16851423/TestGetPictureAbsolutePath.xlsx)

**Describe the results you received:**

The above test fails:

```shell
$ go test -run TestGetPictureAbs
--- FAIL: TestGetPictureAbs (0.00s)
    picture_test.go:26: 
        	Error Trace:	picture_test.go:26
        	Error:      	Not equal: 
        	            	expected: []string{"A1"}
        	            	actual  : []string(nil)        	            
        	            	 
        	Test:       	TestGetPictureAbs
FAIL
exit status 1
FAIL	github.com/xuri/excelize/v2	0.300s
```

**Describe the results you expected:**

Using either relative or absolute paths for relationship targets should be acceptable.

**Output of `go version`:**

```text
go version go1.21.6 darwin/arm64
```

**Excelize version or commit ID:**

excelize master @ 0447cb22c8e9a0fff4e76d4a499df65555a2de29

**Environment details (OS, Microsoft Excel™ version, physical, etc.):**

* macOS 14.6.1
* Microsoft Excel for Mac version 16.88

--- a/picture.go
+++ b/picture.go
@@ -497,8 +497,7 @@ func (f *File) GetPictures(sheet, cell string) ([]Picture, error) {
 	target := f.getSheetRelationshipsTargetByID(sheet, ws.Drawing.RID)
 	drawingXML := strings.TrimPrefix(strings.ReplaceAll(target, "..", "xl"), "/")
 	drawingRelationships := strings.ReplaceAll(
-		strings.ReplaceAll(target, "../drawings", "xl/drawings/_rels"), ".xml", ".xml.rels")
-
+		strings.ReplaceAll(drawingXML, "xl/drawings", "xl/drawings/_rels"), ".xml", ".xml.rels")
 	imgs, err := f.getCellImages(sheet, cell)
 	if err != nil {
 		return nil, err
@@ -526,7 +525,8 @@ func (f *File) GetPictureCells(sheet string) ([]string, error) {
 	target := f.getSheetRelationshipsTargetByID(sheet, ws.Drawing.RID)
 	drawingXML := strings.TrimPrefix(strings.ReplaceAll(target, "..", "xl"), "/")
 	drawingRelationships := strings.ReplaceAll(
-		strings.ReplaceAll(target, "../drawings", "xl/drawings/_rels"), ".xml", ".xml.rels")
+		strings.ReplaceAll(drawingXML, "xl/drawings", "xl/drawings/_rels"), ".xml", ".xml.rels")
+
 	embeddedImageCells, err := f.getImageCells(sheet)
 	if err != nil {
 		return nil, err
@@ -609,8 +609,15 @@ func (f *File) getPicture(row, col int, drawingXML, drawingRelationships string)
 		}
 	}
 	cb2 := func(a *decodeCellAnchor, r *xlsxRelationship) {
-		pic := Picture{Extension: filepath.Ext(r.Target), Format: &GraphicOptions{}, InsertType: PictureInsertTypePlaceOverCells}
-		if buffer, _ := f.Pkg.Load(filepath.ToSlash(filepath.Clean("xl/drawings/" + r.Target))); buffer != nil {
+		var target string
+		if strings.HasPrefix(r.Target, "/") {
+			target = strings.TrimPrefix(r.Target, "/")
+		} else {
+			target = filepath.ToSlash(filepath.Clean("xl/drawings/" + r.Target))
+		}
+
+		pic := Picture{Extension: filepath.Ext(target), Format: &GraphicOptions{}, InsertType: PictureInsertTypePlaceOverCells}
+		if buffer, _ := f.Pkg.Load(target); buffer != nil {
 			pic.File = buffer.([]byte)
 			pic.Format.AltText = a.Pic.NvPicPr.CNvPr.Descr
 			pics = append(pics, pic)
@@ -770,7 +777,14 @@ func (f *File) getPictureCells(drawingXML, drawingRelationships string) ([]strin
 		}
 	}
 	cb2 := func(a *decodeCellAnchor, r *xlsxRelationship) {
-		if _, ok := f.Pkg.Load(filepath.ToSlash(filepath.Clean("xl/drawings/" + r.Target))); ok {
+		var target string
+		if strings.HasPrefix(r.Target, "/") {
+			target = strings.TrimPrefix(r.Target, "/")
+		} else {
+			target = filepath.ToSlash(filepath.Clean("xl/drawings/" + r.Target))
+		}
+
+		if _, ok := f.Pkg.Load(target); ok {
 			if cell, err := CoordinatesToCellName(a.From.Col+1, a.From.Row+1); err == nil && inStrSlice(cells, cell, true) == -1 {
 				cells = append(cells, cell)
 			}

--- a/picture_test.go
+++ b/picture_test.go
@@ -219,6 +219,19 @@ func TestGetPicture(t *testing.T) {
 	cells, err := f.GetPictureCells("Sheet2")
 	assert.NoError(t, err)
 	assert.Equal(t, []string{"K16"}, cells)
+
+	// Try to get picture cells with absolute target path in the drawing relationship
+	rels, err := f.relsReader("xl/drawings/_rels/drawing2.xml.rels")
+	assert.NoError(t, err)
+	rels.Relationships[0].Target = "/xl/media/image2.jpeg"
+	cells, err = f.GetPictureCells("Sheet2")
+	assert.NoError(t, err)
+	assert.Equal(t, []string{"K16"}, cells)
+	// Try to get pictures with absolute target path in the drawing relationship
+	pics, err = f.GetPictures("Sheet2", "K16")
+	assert.NoError(t, err)
+	assert.Len(t, pics, 1)
+
 	assert.NoError(t, f.Close())
 
 	// Test get picture from none drawing worksheet

